card/mod/standard/set/all/links.rb

```diff
@@ -1,81 +1,93 @@
-format do
-  # link is called by web_link, card_link, and view_link
-  # (and is overridden in other formats)
-  def link_to text, pathish, _opts={}
-    href = interpret_pathish pathish
+RESOURCE_TYPE_REGEXP = /^([a-zA-Z][\-+\.a-zA-Z\d]*):/
 
+format :html do
+  def link_to text=nil, opts={}
```

Comment:
yes, will definitely add documentation.

I played with html_options (although I called it format_opts, because we're using the same API in all formats).  It was much harder to unify the API that way.

For example link_to_resource (formerly web_link) takes, in essence, a fully formed path as its first argument. 

The current, obvious pattern is `link_to[_target] [target,] text, opts`.  If we switched that to `link_to[_target], [target,] text, path_opts, format_opts, then we'd either have to have a mostly nonsensical 3rd argument (path_opts) for link_to_resource, or we'd have to break the pattern.

Also, we have a common pattern of passing these arguments around quite a lot, (see button_link, toolbar.rb, etc)  If it's one hash, it ends up being a lot easier and cleaner to pass it around.